src/dotnet/APIView/CloneAPIViewDB/CloneAPIView/CloneAPIViewDB.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Is this needed? I ask because we are trying to avoid moving to .NET 7.0 as 6.0 is LTS.

Reply:
This is just the migration script though. Code will be discarded after use.

Reply:
OK If we don't plan to keep this then this is fine for now.